src/interpreter/executors/instruction.rs

```diff
@@ -247,12 +247,23 @@ where
 
             OpcodeRepr::JNEI => {
                 self.gas_charge(GAS_JNEI)?;
-                self.jump_not_equal_imm(a, b, imm)?;
+                self.jump_not_equal(a, b, imm)?;
             }
 
             OpcodeRepr::JNZI => {
                 self.gas_charge(GAS_JNZI)?;
-                self.jump_not_zero_imm(a, imm)?;
+                self.jump_not_zero(a, imm)?;
+            }
+
+            OpcodeRepr::JMP => {
+                self.gas_charge(GAS_JMP)?;
+                self.jump(a)?;
+            }
+
+            OpcodeRepr::JNE => {
+                self.gas_charge(GAS_JNE)?;
+                dbg!("JNE", a, b, c);
```

Comment:
dbg

Reply:
Removed. There really should be a way to detect this in the CI. This is actually now possible, last bugs where fixed in clippy PR 8500. So maybe we should just deny `clippy::dbg_macro` in CI.

Reply:
Can you file an issue to investigate?

Reply:
Sure, will do. This will probably be a good idea for all Fuel Rust repos, but I'll create an issue here first and then follow up with PRs unless we find reasons not to do so.

Reply:
https://github.com/FuelLabs/fuel-vm/issues/163